e2e/interchaintestv8/chainconfig/chain_config.go

```diff
@@ -28,7 +28,7 @@ func IbcGoChainSpec(name, chainId string) *interchaintest.ChainSpec {
 			Denom:          "stake",
 			GasPrices:      "0.00stake",
 			GasAdjustment:  1.3,
-			EncodingConfig: SDKEncodingConfig(),
+			EncodingConfig: CosmosEncodingConfig(),
```

Comment:
We could still call it SDKEncodingConfig?

Reply:
We could. I'll do that